ntbs-service/wwwroot/css/header.scss

```diff
@@ -128,14 +128,21 @@ body:not(.js-enabled) {
 }
 
 .header-logo {
-    padding: 0 40px 5px 0;
+    padding: 4px 40px 5px 0;

```

Comment:
Because the menu button is positioned absolutely to make the logo align with it I had to add a bit of padding to the top when the logo is smaller